### PR TITLE
Make the install script verify the checksum automatically

### PR DESCRIPTION
**Pull request by pior**
## Why

The install script should not ask the user to compare the checksums, it should let `shasum` do that.

## How

Download the checksum file at the right place and run `shasum -c`, which would warn and fail the whole script in case of a mismatch.

BLOCKED until the next release because we need to the new shasum files introduced by #103

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -1,13 +1,12 @@
 #!/usr/bin/env bash
 set -eu
 
-VERSION="v0.3.0"
+VERSION="v0.4.0"
```
Comment (pior) [inline install.sh]: This version doesn't exist but we need the path fix from #103, which will only be available in the next release.

File: install.sh [hunk 2]
```diff
@@ -55,22 +54,22 @@ instructions() {
...
-    read -p "Correct? [enter]"
+    header "Verify SHA256 checksum"
+    shasum -c "${BINARY}.sha256"
 
     header "Installing to ${DEST}"
-    sudo install "${TMPFILE}" "${DEST}/dad"
+    sudo install "${BINARY}" "${DEST}/dad"
 
-    [[ -e "${TMPFILE}" ]] && unlink "${TMPFILE}"
+    rm -rf "${TMPDIR}"
```
Comment (pior) [inline install.sh]: Should we just leave the tmpdir to avoid doing a `rm -rf` ?

Comment (mlhamel) [reply install.sh]: I think that's quite common to keep those tempdir and anyway something should clean this up at one point automatically no?

Comment (mlhamel) [reply install.sh]: i don't like using `-rf` in a script

Comment (pior) [reply install.sh]: Removed

**Pull request merged**